application/src/main/java/org/thingsboard/server/controller/plugin/TbWebSocketHandler.java

```diff
@@ -144,8 +146,9 @@ public void afterConnectionEstablished(WebSocketSession session) throws Exceptio
             }
             var tenantProfileConfiguration = getTenantProfileConfiguration(sessionRef);
             internalSessionMap.put(internalSessionId, new SessionMetaData(session, sessionRef,
-                    tenantProfileConfiguration != null && tenantProfileConfiguration.getWsMsgQueueLimitPerSession() > 0 ?
-                            tenantProfileConfiguration.getWsMsgQueueLimitPerSession() : 500));
+                    tenantProfileConfiguration != null && tenantProfileConfiguration.getWsMsgQueueLimitPerSession() > 0 &&
```

Comment:
Please put braces around the logical expression. This will make code more readable and eliminate the chance of issues with the order of the operations: 

```
(tenantProfileConfiguration != null && tenantProfileConfiguration.getWsMsgQueueLimitPerSession() > 0 && tenantProfileConfiguration.getWsMsgQueueLimitPerSession() < wsMaxQueueMessagesPerSession) ? tenantProfileConfiguration.getWsMsgQueueLimitPerSession() : wsMaxQueueMessagesPerSession
```

For the best clarity of the code, you should extract the ```tenantProfileConfiguration.getWsMsgQueueLimitPerSession()``` to a separate variable, for example: ```wsTenantProfileQueueLimit```